src/sx/src/css-properties/isUnitlessNumber.js

```diff
@@ -41,7 +31,6 @@ export default {
   orphans: true,
   tabSize: true,
   widows: true,
-  zIndex: true,
```

Comment:
Can you please revert all the changes in this file? It should not be changed - all the values there are correct (check the description).

The proper way how to fix this should be to remove this logic here: https://github.com/adeira/universe/blob/a88d1102c5c0c26b97df984ffd5e29534ca922d5/src/sx/src/css-properties/generate.js#L27 (keep there only `string | number` even though it's less precise). It's incorrect to confuse unitless numbers with `number` type. My mistake in the generator. 🙈